src/components/Meetings.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import { Bars } from 'react-loader-spinner'
+import { useEffect, useState } from "react"
+import { useParams, Link } from "react-router-dom"
+
+export default function Meetings() {
```

Comment:
This is the last thing to review, Your code is very clean and well structured, and I like the way u did for loops as they might be more comfortable to use, but after u finish the logic of your program do some refactoring and consider using more advanced methods like map and objects. (this is just an opinion it might be wrong)